src/main/java/org/example/db/DeliveryEmployeeDao.java

```diff
@@ -60,4 +60,21 @@ public int createDeliveryEmployee(DeliveryEmployeeRequest employee) throws SQLEx
 
         return -1;
     }
+
+    public DeliveryEmployee getDeliveryEmployeeById(int id) throws SQLException {
+        Connection c = databaseConnector.getConnection();
+        Statement st = c.createStatement();
+        ResultSet rs = st.executeQuery("SELECT delivery_employee_id, `name`, salary, bank_account_number, national_insurance_number FROM DeliveryEmployee WHERE delivery_employee_id="+id);
```

Comment:
use a prepared statement to avoid SQL injection

Reply:
Updated.